libs/synapse/api-description/build/openapi.yaml

```diff
@@ -0,0 +1,76 @@
+openapi: 3.0.3
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+  schemas:
+    ChallengePagedResults:
+      title: ChallengePagedResults
+      type: object
+      properties:
+        totalNumberOfResults:
+          type: integer
+          description: The total number of results
+      description: List the Challenges for which the given participant is registered. To be in the returned list the caller must have READ permission on the project associated with the Challenge.
+security:
+  - bearerAuth: []
+info:
+  title: Synapse REST API
+  version: 0.1.0
+  license:
+    name: Apache 2.0
+    url: https://github.com/Sage-Bionetworks/sage-monorepo
+  contact:
+    name: Support
+    url: https://github.com/Sage-Bionetworks/sage-monorepo
+  x-logo:
+    url: https://Sage-Bionetworks.github.io/rocc-schemas/logo.png
```

Comment:
Nit:  Should this be different?

Reply:
I removed info properties that are not strictly required for the sake of keeping the example minimal.